executor/aggfuncs/func_count.go

```diff
@@ -0,0 +1,375 @@
+package aggfuncs
+
+import (
+	"encoding/binary"
+	"unsafe"
+	"github.com/juju/errors"
```

Comment:
We usually separate the imported packages to two sections: one for the golang standard packages, one for the third part packages. For example: https://github.com/pingcap/tidb/blob/master/executor/aggfuncs/aggfuncs.go#L16. So here we should put **"github.com/juju/errors"** together with other third part packages and leave an empty line between the golang standard packages like **"unsafe"** 😄

Reply:
Well, I misread the comment: https://github.com/pingcap/tidb/pull/7009#discussion_r201751300 , I will fix it.